systems/subscriber/sim-manager/pkg/server/sim_manager.go

```diff
@@ -252,6 +262,22 @@ func (s *SimManagerServer) GetSimsBySubscriber(ctx context.Context, req *pb.GetS
 	return resp, nil
 }
 
+func (s *SimManagerServer) GetSimCounts(ctx context.Context) error {
+	terminatedCount, simsCount, activeCount, deactiveCount, err := s.simRepo.GetSimCounts()
+	if err != nil {
+		log.Errorf("failed to get Sims counts: %s", err.Error())
+		return err
+	}
+	customLabels := map[string]string{
```

Comment:
Kind of confused here why the Metric/KPI data is set as label here ? Active/Inactive/Terminated should be a metric on its own, not label.

Reply:
I have updated the Pushgateway util function,  now we are able to retrieve metrics for all KPIs instead of just labels.